docs/2_1_common_api.adoc

```diff
@@ -65,6 +65,7 @@ In order for this to be possible,
 the names of the functions in different FMUs must be different, or function pointers must be used.
 To support the first variant macros are provided in `"fmi2Functions.h"` to build the actual
 function names by using a function prefix that depends on how the FMU is shipped.
+_[These macros can be modified to compile the shared library from the source code e.g. to simulate an FMU that has been exported on a different platform.]_
 Typically, FMU functions are used as follows:
```

Comment:
Can't we explicitly write, where these FMU functions are used? Is the C-Code defining the FMU?